src/clients/users_auto.rb

```diff
@@ -171,6 +171,8 @@ def check_users(users)
       if users.size > users.uniq { |u| u["username"]}.size
         Report.Error(_("Found users in profile with equal <username>."))
       end
+      # Do not check users without defined UID. (bnc#996823)
+      users.reject! { |u| !u.key?("uid")}
```

Comment:
it modifies input parameter, is it OK? And if so, then please document it in method definition, but such nasty side effect should be avoided.